### PR TITLE
Make it easier to understand the type mapping between KT and JS

### PR DESCRIPTION
**Pull request by lppedd**
This change replaces the bullet point list with a table, which is easier to follow and understand.

### CODE REVIEW

File: docs/topics/js/js-to-kotlin-interop.md [hunk 1]
```diff
@@ -120,9 +120,34 @@ the JavaScript target, and allows you to also export Kotlin declarations that ar
...
+| `LongArray`                               | `Array<kotlin.Long>`        | Carries the property `$type$ == "LongArray"`. Also see Kotlin's Long type comment.         |
+| `BooleanArray`                            | `Int8Array`                 | Carries the property `$type$ == "BooleanArray"`                                            |
+| `Unit`                                    | Undefined                   |                                                                                            |
+| `Any`                                     | `Object`                    |                                                                                            |
+| `Throwable`                               | `Error`                     |                                                                                            |
+| Nullable `Type?`                          | `Type \| null \| undefined` |                                                                                            |
+| All other Kotlin types                    | Not supported               | Includes Kotlin's collections (`List`, `Set`, `Map`, etc.), and unsigned variants.         |
```
Comment (anton-bannykh) [inline docs/topics/js/js-to-kotlin-interop.md:145]: Except for those marked with `JsExport` annotation

**Review by anton-bannykh — APPROVED** [review]
Looks about right

**Review by koshachy — APPROVED** [review]
I apologize for the delayed response. 

Thank you for your contribution to the Kotlin documentation!

**Comment by koshachy** [comment]
@lppedd 
Yes, I've double-checked it at staging before merging :) 
It is in production now and looks good!

Thank you again!

**Pull request merged**